schema/index.js

```diff
@@ -46,9 +46,9 @@ const schema = new GraphQLSchema({
       artworks: Artworks,
       artist: Artist,
       artists: Artists,
+      external_partner: ExternalPartner,
       fair: Fair,
       fairs: Fairs,
-      galaxy_partner: GalaxyPartner,
```

Comment:
FYI we don’t use the `partner` root field in Eigen atm, so it will be easy to convert the `partner` field to a union.